spec/requests/graphql_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+RSpec.describe("v1.0 - GraphQL") do
+
+  around do |example|
+    bypass_rbac do
+      example.call
+    end
+  end
+
+  let(:tenant)      { create(:tenant) }
+  let(:portfolio_a) { create(:portfolio, :tenant_id => tenant.id, :description => 'test a desc', :name => "test_a", :owner => "234") }
+  let(:portfolio_b) { create(:portfolio, :tenant_id => tenant.id, :description => 'test b desc', :name => "test_b", :owner => "123") }
+
+  let(:graphql_portfolio_query) { { "query" => "{ portfolios { edges { node { id name } } } }" } }
+
+  def result_portfolio_ids(response_body)
+    JSON.parse(response_body).fetch_path("data", "portfolios", "edges").collect { |edge| edge.fetch_path("node", "id").to_i }
+  end
+
+  def result_portfolio_names(response_body)
+    JSON.parse(response_body).fetch_path("data", "portfolios", "edges").collect { |edge| edge.fetch_path("node", "name") }
+  end
+
+  context "different graphql queries" do
+    before do
+      portfolio_a
+      portfolio_b
```

Comment:
You should just be able to do `let!` on both `portfolio_a` and `portfolio_b` to avoid having to call these here.